zetta_utils/training/lightning/train.py

```diff
@@ -63,3 +83,69 @@ def lightning_train(
         val_dataloaders=val_dataloader,
         ckpt_path=ckpt_path,
     )
+
+
+@builder.register("lightning_train_remote")
+@typeguard.typechecked
+def lightning_train_remote(image: str, resources: dict, spec_path: str) -> None:
```

Comment:
User should be able to choose the cluster rather than falling back on default. We often juggle clusters and projects, and it's important to be able to do that without code chages.